indico/modules/events/registration/schemas.py

```diff
@@ -0,0 +1,14 @@
+from marshmallow.fields import Integer, String
+
+from indico.core.marshmallow import mm
+from indico.modules.events.registration.models.forms import RegistrationForm
+
+
+class RegistrationFormSchema(mm.ModelSchema):
+    class Meta:
+        model = RegistrationForm
+        fields = ('id', 'name', 'identifier')
+
+    id = Integer(attribute='id')
+    name = String(attribute='title')
+    identifier = 'RegistrationForm:{}'.format(id)
```

Comment:
I don't think this works...

Reply:
But you don't need any of the last three lines, because they are all attributes of RegistrationForm and should be handled implicitly. In case `identifier` (not a column) isn't, then `identifier = String()` would be enough.

Reply:
It seemed to work, but I did not notice there was already a property in the model. I was trying to avoid adding a new one solely for the principal list purpose. Thanks!